wikichange/src/content/content.js

```diff
@@ -259,9 +258,38 @@ const renderItemsBelowGraph = async (creationDate) => {
             getRevisionPageLink(title, curRevisionId, oldRevisionId).replace(/\s/g, "_"),
             oldRevisionDate
         );
-
         highlightRevisionBetweenRevisionIds(title, curRevisionId, oldRevisionId);
     });
+    return [curRevisionId, oldRevisionId];
+};
+
+/**
+ * Render a simple JS loader by the highlight button
+ */
+const renderLoader = () => {
+    let button = document.getElementById("highlightButton");
+    button.disabled = true;
+
+    let loader = document.getElementById("loader");
+    loader.style.paddingTop = "3px";
+    loader.style.border = "5px solid white";
+    loader.style.borderTop = "5px solid #3498db";
+    loader.style.borderRadius = "50%";
+    loader.style.width = "15px";
+    loader.style.height = "15px";
+    loader.style.position = "absolute";
+    loader.style.marginTop = "6.5px";
+    loader.style.marginLeft = "350px";
```

Comment:
```suggestion
    loader.style.marginLeft = "335px";
```
To make it closer 
<img width="604" alt="image" src="https://user-images.githubusercontent.com/55929961/218296799-38401742-e578-432e-9b6b-bb8628af1e24.png">